logs/logic1. log

```diff
@@ -0,0 +1,398 @@
+2021-04-27T11:17:58.582Z        DEBUG   provider-template       Reconciling     {"controller": "managed/nopresource.sample.template.crossplane.io", "request": "/example"}
+
+
+        My values
+
+14.58277647s
+
+ReconcileSuccess  True Synced
+
+Available  True Ready
+
+2021-04-27T11:17:58.582Z        DEBUG   provider-template       External resource is up to date {"controller": "managed/nopresource.sample.template.crossplane.io", "request": "/example", "uid": "94b97e85-99ff-4b3c-8f2d-fc81ae3e3716", "version": "1498856", "external-name": "example", "requeue-after": "2021-04-27T11:17:59.582Z"}
+2021-04-27T11:17:59.588Z        DEBUG   provider-template       Reconciling     {"controller": "managed/nopresource.sample.template.crossplane.io", "request": "/example"}
+Calling update on index 2
+
+
+        My values
+
+15.588178778s
+
+Available  False Synced
+
+Available  True Ready
+
+2021-04-27T11:17:59.588Z        DEBUG   provider-template       External resource is up to date {"controller": "managed/nopresource.sample.template.crossplane.io", "request": "/example", "uid": "94b97e85-99ff-4b3c-8f2d-fc81ae3e3716", "version": "1498856", "external-name": "example", "requeue-after": "2021-04-27T11:18:00.588Z"}
+2021-04-27T11:17:59.593Z        DEBUG   provider-template       Reconciling     {"controller": "managed/nopresource.sample.template.crossplane.io", "request": "/example"}
+Calling update on index 2
+
+
+        My values
+
+15.593518279s
+
+Available  False Synced
+
+Available  True Ready
+
+2021-04-27T11:17:59.593Z        DEBUG   provider-template       External resource is up to date {"controller": "managed/nopresource.sample.template.crossplane.io", "request": "/example", "uid": "94b97e85-99ff-4b3c-8f2d-fc81ae3e3716", "version": "1498873", "external-name": "example", "requeue-after": "2021-04-27T11:18:00.593Z"}
+2021-04-27T11:18:00.593Z        DEBUG   provider-template       Reconciling     {"controller": "managed/nopresource.sample.template.crossplane.io", "request": "/example"}
+
+
+        My values
+
+16.593581987s
+
+ReconcileSuccess  True Synced
+
+Available  True Ready
+
+2021-04-27T11:18:00.593Z        DEBUG   provider-template       External resource is up to date {"controller": "managed/nopresource.sample.template.crossplane.io", "request": "/example", "uid": "94b97e85-99ff-4b3c-8f2d-fc81ae3e3716", "version": "1498873", "external-name": "example", "requeue-after": "2021-04-27T11:18:01.593Z"}
+2021-04-27T11:18:01.599Z        DEBUG   provider-template       Reconciling     {"controller": "managed/nopresource.sample.template.crossplane.io", "request": "/example"}
+
+
+        My values
+
+17.599209698s
+
+ReconcileSuccess  True Synced
```

Comment:
The major difference in the 2 behaviors can be seen after 15s.

Reply:
The output you are seeing above is `fmt.Printf("%s %s %s %s", string(e.Reason), string(e.Message), string(e.Status), string(e.Type))`